db/migrate/20150611083448_change_custom_attribute_value_type_to_text.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeCustomAttributeValueTypeToText < ActiveRecord::Migration
+  def up
+    change_column :custom_attributes, :value, :text
```

Comment:
@blomquisg ah, we chatted about this, he thought I am creating new table here. So you can ignore this comment :-)